benchmarks/README.md

```diff
@@ -0,0 +1,75 @@
+# Benchmarking SpatialData code
+
+This `benchmarks/` folder contains code to benchmark the performance of the SpatialData code. You can use it to see how code behaves for different options or data sizes. For more information, check the [SpatialData Contributing Guide](https://spatialdata.scverse.org/en/stable/contributing.html).
+
+Note that to run code, your current working directory should be the SpatialData repo, not this `benchmarks/` folder.
+
+## Installation
+
+The benchmarks use the [airspeed velocity](https://asv.readthedocs.io/en/stable/) (asv) framework. Install it with the `benchmark` option:
+
+```
+pip install -e '.[docs,test,benchmark]'
+```
+
+## Usage
+
+Running all the benchmarks is usually not needed. You can select the benchmarks your interesting in with `-b` and run them in your current environment with `--python=same`. Some example benchmarks:
```

Comment:
I'd explain that -b is a regexp, and add the link to the file where the function is located